src/components/TaskTracker/Task.tsx

```diff
@@ -94,9 +117,42 @@ export const Task = ({ task }) => {
             </div>
           </div>
         </div>
+
       ) : (
         <Settings setOpenSettings={setOpenSettings} Task={task} />
       )}
+      <div
+        className="absolute">
+        {task.menuToggled && (
+          <div
+            className="bg-neutral-800 rounded-md" onMouseLeave={closeOnBoundsExit}>
+            <ul className="w-full">
+              <li
+                onClick={() => { toggleInProgressState(task.id) }}
+                className="px-5 py-2 hover:bg-neutral-600 rounded-md">
+                <div>
+                  Select Task
```

Comment:
Instead of Select Task - can we do Track Task instead?

Reply:
Sure! No problems there, I was having a hard time figuring out what to name it.

Resolved in c9b6320109809b08f43a9f577ed4c102ddc1146